sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/_base.py

```diff
@@ -67,6 +68,7 @@ def __init__(self, **kwargs: Any) -> None:
         parsed_connection_string = ConnectionStringParser(kwargs.get('connection_string'))
 
         self._api_version = kwargs.get('api_version') or _SERVICE_API_LATEST
+        self._credential = kwargs.get('credential')
```

Comment:
Be sure to add this to the docstring of the constructor.

Reply:
Done